README.md

```diff
@@ -361,17 +361,31 @@ sign_in_as(user)
 sign_out
 ```
 
-## Credits
+## Contributing
 
-![thoughtbot](http://thoughtbot.com/images/tm/logo.png)
+Please see [CONTRIBUTING.md].
+Thank you, [contributors]!
 
-Clearance is maintained by [thoughtbot, inc](http://thoughtbot.com/community)
-and [contributors](https://github.com/thoughtbot/clearance/contributors) like
-you. Thank you!
+[CONTRIBUTING.md]: /CONTRIBUTING.md
+[contributors]: https://github.com/thoughtbot/clearance/graphs/contributors
 
 ## License
 
-Clearance is copyright © 2009 thoughtbot. It is free software, and may be
-redistributed under the terms specified in the `LICENSE` file.
+Clearance is copyright © 2009-2015 thoughtbot. It is free software, and may be
```

Comment:
There was a conversation in slack in early January about copyright years where it was deemed that we didn't need to include the range. That conversation lead to this change: https://github.com/thoughtbot/clearance/commit/d372ebd3adbb646dea2822c25d1530da5bc36c5f

https://thoughtbot.slack.com/archives/general/p1420728595009700